packages/baggage-span-processor/README.md

```diff
@@ -0,0 +1,63 @@
+# OpenTelemetry Baggage Span Processor for Node.js
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-url]
+
+The BaggageSpanProcessor reads entries stored in Baggage from the parent context and adds the [baggage](https://opentelemetry.io/docs/concepts/signals/baggage) entries' keys and
+values to the span as attributes on span start.
+
+Add this span processor to a tracer provider.
+
+Keys and values added to Baggage will appear on subsequent child spans for a trace within this service *and* be propagated to external services in accordance with any configured propagation formats configured.
+If the external services also have a Baggage span processor, the keys and values will appear in those child spans as well.
+
+⚠ Warning ⚠️
+
+Do not put sensitive information in Baggage.
+
+To repeat: a consequence of adding data to Baggage is that the keys and values will appear in all outgoing HTTP headers from the application.
+
+Compatible with OpenTelemetry JS API and SDK `1.0+`.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/baggage-span-processor
+```
+
+### Usage
+
+Add to the span processors during configurtion:
+
+```javascript
+import { SemanticResourceAttributes } from "@opentelemetry/semantic-conventions";
+import { NodeSDK } from "@opentelemetry/sdk-node";
+import { BaggageSpanProcessor } from "@opentelemetry/baggage-span-processor";
+
+const globalResource = new Resource({
+   [SemanticResourceAttributes.SERVICE_NAME]: "TestService",
+});
+
+const sdk = new NodeSDK({
+   resource: globalResource,
+   SemanticResourceAttributes: new BaggageSpanProcessor(),
+});
+
+sdk.start()
```

Comment:
Looks like a typo on span processor, also NodeSDK has updated to include a few extra things that can simplify this a bit further. Maybe something like this:

```suggestion
import { NodeSDK, tracing } from "@opentelemetry/sdk-node";
import { BaggageSpanProcessor } from "@opentelemetry/baggage-span-processor";

const spanProcessors = [
  new tracing.SimpleSpanProcessor(
    new tracing.ConsoleSpanExporter()),
  new BaggageSpanProcessor()];

const sdk = new NodeSDK({
  serviceName: "example-service",
  spanProcessors
});

sdk.start();
```